src/modules/cmdpal/exts/Microsoft.CmdPal.Ext.Apps/Commands/RunAsUserCommand.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.Apps.Utils;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.Ext.Apps.Commands;
+
+// NOTE this is pretty close to what we'd put in the SDK
+internal sealed partial class RunAsUserCommand : InvokableCommand
+{
+    private readonly string _target;
+    private readonly string _parentDir;
+
+    public RunAsUserCommand(string target, string parentDir)
+    {
+        Name = "Run as different user";
```

Comment:
does this one need to get localized too?

Reply:
ooh, nice catch. there are few more. fixed